src/components/BuyButton.tsx

```diff
@@ -3,40 +3,27 @@ import { loadStripe } from "@stripe/stripe-js";
 import {
   SpaceProps,
   LayoutProps,
-  FlexboxProps,
-  PositionProps,
-  ColorProps,
   TypographyProps,
   BorderProps,
+  BackgroundProps,
   space,
   layout,
-  color,
   typography,
-  flexbox,
   border,
-  position,
+  background,
 } from "styled-system";
 import styled from "styled-components";
 // Make sure to call `loadStripe` outside of a component’s render to avoid
 // recreating the `Stripe` object on every render.
 
-type LogoProps = SpaceProps &
-  LayoutProps &
-  FlexboxProps &
-  PositionProps &
-  ColorProps &
-  SpaceProps &
-  TypographyProps &
-  BorderProps;
-
-const Button = styled.button<LogoProps>`
+const Button = styled.button<
+  SpaceProps & TypographyProps & BorderProps & BackgroundProps
+>`
+  width: max-content;
```

Comment:
Doesn't work on ie11

What problem was this fixing?

Could use: 
```
width: auto;
white-space: nowrap;
```

Orrrr poss better, `display: flex`?

Reply:
Basically I was trying to style everything that is seen on the home page including the buy button component - granted can understanding why this is confusing

Reply:
The reason for no `display: flex` was this

<img width="107" alt="Screenshot 2020-07-20 at 19 16 04" src="https://user-images.githubusercontent.com/53219789/87966477-9b382880-cabd-11ea-906d-b25fb1f07eb0.png">

but `width: auto;
white-space: nowrap;` is a winner